22/alpine3.18/Dockerfile

```diff
@@ -0,0 +1,112 @@
+FROM alpine:3.18
+
+ENV NODE_VERSION 20.0.0
```

Comment:
these all say 20, not 22

Reply:
Sorry that's my mistake! Fixed with 6a392a8ca911481f956c186654ceabd327fcbb49